tyk-stack/README.md

```diff
@@ -1,11 +1,12 @@
-## Tyk single dc
+## Tyk Stack
 
-`tyk-stack` provides the default deployment of Tyk Pro on single data center. It will deploy all required Tyk components with the settings provided in the values.yaml file.
+`tyk-stack` provides the default deployment of Tyk Self Managed on a cluster. It will deploy all required Tyk components with the settings provided in the values.yaml file.
```

Comment:
Same as https://github.com/TykTechnologies/tyk-charts/pull/152/files#r1377226779. If it makes sense for you, we can update them. Otherwise, please ignore my comment.
```suggestion
`tyk-stack` provides the default deployment of Tyk Self Managed on a cluster. It will deploy all required Tyk components with the settings provided in the `values.yaml` file.
```